src/session-summaries/understanding-browser-rendering.md

```diff
@@ -0,0 +1,22 @@
+---
+
+hosts: Segun Ola, Enodi Audu
+guests:
+summarized_by: Chimdia Anyiam
+summarized_by_social_media: https://twitter.com/AnyiamChimdia
+
+---
+
+# Understanding browser rendering
+
+This session has Enodi Audu introducing the concept of browser rendering and steps that the browsers take to convert our HTML code to what we can see on the screen.
+
+The browser receives raw bytes of data that it does not understand, these bytes are then converted to characters depending on the character encoding specified in the HTML head tag e.g. UTF-8. These characters are tokenized and the tokens are converted to nodes which are likned in tree data structure known as the Document Object Model (DOM). The host does well too explain this in more details.
```

Comment:
This is concise and clear. Thanks.

Can you change the last sentence from _The host does well `too`_ to _Enodi does well `to`_